file 6

```diff
@@ -0,0 +1,9 @@
+fasdfad
+fa
+sdf
+ads
```

Comment:
hello lines 3-7
 - [1](https://github.com/drok/octokit.net/pull/5/files/b4d189acc450d4ab38b16a29c7eb38ffa3e7a8db#r936052643)
 - [2](https://github.com/drok/octokit.net/pull/5/files/b4d189acc450d4ab38b16a29c7eb38ffa3e7a8db#r936051891)

Reply:
> hello lines 3-7
> 
> * [1](https://github.com/drok/octokit.net/pull/5/files/b4d189acc450d4ab38b16a29c7eb38ffa3e7a8db#r936052643)
> * [2](https://github.com/drok/octokit.net/pull/5/files/b4d189acc450d4ab38b16a29c7eb38ffa3e7a8db#r936051891)

Is commit 9dc3f61 a bugfix or new feature?
 - [ ]bugfix
 - [ ] new feature

Reply:
> > hello lines 3-7
> > 
> > * [1](https://github.com/drok/octokit.net/pull/5/files/b4d189acc450d4ab38b16a29c7eb38ffa3e7a8db#r936052643)
> > * [2](https://github.com/drok/octokit.net/pull/5/files/b4d189acc450d4ab38b16a29c7eb38ffa3e7a8db#r936051891)
> 
> Is commit [9dc3f61](https://github.com/drok/octokit.net/commit/9dc3f61c69920a3f33f67ad1188c1c3c6e9b573c) a bugfix or new feature?
> 
> * [ ]  bugfix
> * [ ]  new feature
> 
> Is it related to #3 ?
> 
> * [ ]  yes
> * [ ]  no

Reply:
> hello lines 3-7
> 
> * [1](https://github.com/drok/octokit.net/pull/5/files/b4d189acc450d4ab38b16a29c7eb38ffa3e7a8db#r936052643)
> * [2](https://github.com/drok/octokit.net/pull/5/files/b4d189acc450d4ab38b16a29c7eb38ffa3e7a8db#r936051891)

New comment without being collaborator:
Is it related to Is commit 9dc3f61 a bugfix or new feature?
 - [ ] bugfix
 - [ ] new feature